ethdb/pebble/pebble.go

```diff
@@ -125,7 +125,13 @@ func New(file string, cache int, handles int, namespace string, readonly bool) (
 		handles = minHandles
 	}
 	logger := log.New("database", file)
-	logger.Info("Allocated cache and file handles", "cache", common.StorageSize(cache*1024*1024), "handles", handles)
+	logger.Info(
+		"Allocated cache and file handles",
+		"cache",
+		common.StorageSize(cache*1024*1024),
+		"handles",
+		handles,
+	)
```

Comment:
Can you revert this change? I think the original format is better.

Reply:
Yep. Sorry, my linter has a mind of its own.